app/resources/forms.js.template

```diff
@@ -375,4 +375,14 @@ function make_form_expandable(expandedSection) {
   });
 }
 
+// Show the side menu
+function menuOpenNav() {
+  document.getElementById("mySidenav").style.width = "250px";
+  document.getElementById("overLay").style.width = "100%";
```

Comment:
It would be more natural to change visibility of this element than changing width.